cmds/xbmc.py

```diff
@@ -710,7 +719,7 @@ def _files(bot,args,dirs,pid):
   # set last_played for bookmarking
   bot.last_played = (pid,matches[0])
 
-  return msg+'Playlist from "'+match+'" starting with #'+str(num+1)
+  return msg+'Playlist starting with #'+str(num+1)
```

Comment:
I'm wary of not including the path in the response, as malformed searches can yield unexpected results. Perhaps just add a config option e.g. library.respond_path or something?